pages/use-solid/apps.md

```diff
@@ -6,111 +6,145 @@ redirect_from:
-* [Launcher Exploration](https://launcher-exploration.inrupt.app/) (code on https://github.com/inrupt/launcher-exploration)
-* [Poddit](https://vincenttunru.gitlab.io/poddit/). 2019. Vincent Tunru.
-* [Solid authorization Widget](https://bourgeoa.solidcommunity.net/public/solid-file-widget/) component for web app. [source code](https://github.com/bourgeoa/solid-file-widget) is under [MIT License Copyright (c) 2019 ](https://github.com/bourgeoa/solid-file-widget/blob/master/LICENSE) by [Bourgeoa](https://github.com/bourgeoa/solid-file-widget/blob/master/LICENSE)
-* [OpenLink Smart Data Bot (OSDB)](http://osdb.openlinksw.com) distills actions from API documentation constructed using RDF or OpenAPI; supports WebID-OIDC for authentication (c) 2019 [OpenLink Software](http://www.openlinksw.com)
-* [Solid Shell](https://github.com/jeff-zucker/solid-shell) command-line tool and interactive shell. [Source code](https://github.com/jeff-zucker/solid-shell) [MIT License Copyright (c) 2019](https://github.com/jeff-zucker/solid-shell/blob/master/LICENSE) [Jeff Zucker](https://github.com/jeff-zucker)
-* [Solid File Widget](https://bourgeoa.solidcommunity.net/public/solid-file-widget/) widget authorisation. [source code](https://github.com/bourgeoa/solid-file-widget) is under [MIT License Copyright (c) 2019](https://github.com/bourgeoa/solid-file-widget/blob/master/LICENSE) by [Alain Bourgeois](https://github.com/bourgeoa)
-* [Solid IDE](https://jeff-zucker.github.io/solid-ide/) file manager and IDE. [Source code](https://github.com/jeff-zucker/solid-ide) [MIT License Copyright (c) 2018](https://github.com/jeff-zucker/solid-ide/blob/master/LICENSE) [Jeff Zucker](https://github.com/jeff-zucker)
-* [graphMetrix](https://graphmetrix.net/#/) allows you to browse your Solid Pod offering multiple views of information including overview, graph, doc, gallery and grid as well as easy to use Solid collaboration control and file management. 30 day free trial followed by a $10 per user/per month. 2018 [graphMetrix](https://graphmetrix.com/#/solid)
-* LinkedPipes Applications create your own visualisers based on linked data. [Source code](https://github.com/linkedpipes/applications) [Apache License 2.0 (c) 2018](https://github.com/linkedpipes/applications/blob/master/LICENSE) [Jakub Klimek](https://github.com/jakubklimek)
-* [SPARQL Fiddle](https://jeff-zucker.github.io/sparql-fiddle/) online fiddle to run SPARQL against  Pods. [Source code](https://github.com/jeff-zucker/sparql-fiddle) [MIT License Copyright (c) 2018](https://github.com/jeff-zucker/sparql-fiddle/blob/master/LICENSE) [Jeff Zucker](https://github.com/jeff-zucker)
-* [Warp](https://linkeddata.github.io/warp/) file browser. [Source code](https://github.com/linkeddata/warp) MIT License Copyright (c) 2014](https://github.com/linkeddata/warp/blob/gh-pages/LICENSE) [Andrei Samba](https://github.com/deiu)
+
```

Comment:
Kind of feel like Pod Management apps should be up near the top of this page instead of being after "Others" section near the bottom.